scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1021,6 +1022,49 @@
 
 (def pool-name->unmatched-job-uuid->unmatched-cycles-atom (atom {}))
 
+(counters/defcounter [cook-scheduler match cycle-considerable])
+(counters/defcounter [cook-scheduler match cycle-matched])
+(counters/defcounter [cook-scheduler match cycle-unmatched])
+
+(defn handle-match-cycle-metrics
+  [pool-name max-considerable considerable-jobs matches offers offers-scheduled head-job-matched? head-job-resources
```

Comment:
I assume head-job-matched and head-job-resources are also used in the other code and can't be moved entirely here?

Reply:
Also, to me, it seems like we should consider passing these as a dictionary and destructure.  (I.e., this is how clojure does keyword arguments) Doing it this way seems fragile. Get a field out of order and the answers are wrong.

Thoughts?

Reply:
Yes, the head-matched stuff is used to tune Fenzo later in the fn. And yes, can do the destructuring!